class-tgm-plugin-activation.php

```diff
@@ -2023,6 +2039,7 @@ public function force_activation() {
 					} elseif ( $this->can_plugin_activate( $slug ) ) {
 						// There we go, activate the plugin.
 						activate_plugin( $plugin['file_path'] );
+						$this->force_activated_plugins[] = $slug;
```

Comment:
Thanks for the thoughtful feedback. I like all of these ideas, both from
a UX and engineering perspective. I'm aiming to get to these changes and
additions in the next week or two and will update the pull request.